plugins-compat-tester-cli/pom.xml

```diff
@@ -26,6 +25,7 @@
                   <goal>shade</goal>
                 </goals>
                 <configuration>
+                  <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
Trying to work around https://github.com/mojohaus/flatten-maven-plugin/issues/100.